deploy/test/helm-props/k8s_secrets_two.yml

```diff
@@ -0,0 +1,9 @@
+# Not to be shipped. Created for end-to-end testing
+apiVersion: v1
+kind: Secret
+metadata:
+  name: test-k8s-secret-two
```

Comment:
@sigalsax what's the purpose of this k8s secret? can't we use the one that we already have?